.github/workflows/dhis2-preview-pr.yml

```diff
@@ -24,12 +24,12 @@ jobs:
             - uses: actions/checkout@v2
             - uses: actions/setup-node@v1
               with:
-                  node-version: 14.x
+                  node-version: 18.x
```

Comment:
Maybe v20 would be better as that's the currently active LTS: https://nodejs.org/en/about/previous-releases

Reply:
Also, if we're bumping node versions we might also update the .nvmrc in the root with that version. Just to keep things consistent (and that goes for other areas where we're specifying a node version as well).